core/src/main/java/org/testcontainers/utility/LogUtils.java

```diff
@@ -39,4 +42,20 @@ public void followOutput(DockerClient dockerClient, String containerId,
     public void followOutput(DockerClient dockerClient, String containerId, Consumer<OutputFrame> consumer) {
         followOutput(dockerClient, containerId, consumer, STDOUT, STDERR);
     }
+
+    public static String logSafe(AuthConfig authConfig) {
```

Comment:
I'm not sure if this is an appropriate place to put this logging, AFAIK `LogUtils` was about dealing with stdout/stderr of Docker containers

Also, it doesn't actually log, rather generates `toString` for `AuthConfig` objects

Reply:
The perils of utility classes 😭 

Really, an extension method for `AuthConfig` (or an upstream patch) would be ideal.

How about I move it to another utility class and rename?

Reply:
sounds ok to me 👍